### PR TITLE
feat: #2616 migrate button to elements v2

### PR DESCRIPTION
**Pull request by vuhuucuong**
# Pull request checklist

## Does this close any currently open issues?

fixes: #2616

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #2616

## What is the new behavior?

- Add `Button` and `ButtonGroup` elements
image

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/elements-next/src/components/button/button.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+route: /button
+---
+
+import {  Props } from 'docz'
+import { Button, ButtonGroup } from './button'
+import DocsWrapper from '@/docz/docs-wrapper'
+import Playground from '@/components/playground'
+
+# Button
+## Document
```
Comment (nphivu414) [inline packages/elements-next/src/components/button/button.mdx:12]: Please change to `## Properties`

File: packages/elements-next/src/components/button/button.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+name: Button
+route: /button
+---
+
+import {  Props } from 'docz'
+import { Button, ButtonGroup } from './button'
+import DocsWrapper from '@/docz/docs-wrapper'
+import Playground from '@/components/playground'
+
+# Button
```
Comment (nphivu414) [inline packages/elements-next/src/components/button/button.mdx:11]: Please add a description for Button
```
# Button
The button is an essential element of any design. It's meant to look and behave as an interactive element of your page.
```

File: packages/elements-next/src/components/button/button.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+interface ButtonProps
+  extends React.DetailedHTMLProps<React.ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement> {
+  type?: 'submit' | 'reset' | 'button'
+  variant?: 'primary' | 'secondary' | 'danger' | 'info'
+  isLoading?: boolean
+  isFullWidth?: boolean
+  onClick?: () => void
+  className?: string
+  children?: React.ReactNode
```
Comment (nphivu414) [inline packages/elements-next/src/components/button/button.tsx:23]: No need to define `children` prop here

**Review by nphivu414 — CHANGES_REQUESTED** [review]

**Review by tanphamhaiduong — APPROVED** [review]
LGTM

**Review by nphivu414 — APPROVED** [review]

**Pull request merged**